SR2020/2020-03-05-coronavirus.md

```diff
@@ -0,0 +1,21 @@
+---
+to: sr2020-teams
+subject: Update on Coronavirus and SR
+---
+
+# Update on SR and the COVID-19 Coronavirus
+
+Hello everyone, 
+
+TL;DR: We're not intending to cancel the event, but will be monitoring the situation closely and will be taking action both during and before the event to minimize risk.
```

Comment:
I wonder whether this line is needed? It's a pretty short email as it is, and if anything it's a good idea for teams to read the whole thing anyway

Reply:
Yeah, don't put a TL;DR in a team facing e-mail. We want them to be reading the whole e-mail! Informally between volunteers is one thing, externally facing is another.